Sources/AWSLambdaRuntimeCore/LambdaRuntimeClient.swift

```diff
@@ -22,7 +22,7 @@ import NIOHTTP1
 /// * /runtime/invocation/error
 /// * /runtime/init/error
 extension Lambda {
-    internal struct RuntimeClient {
+    struct RuntimeClient {
```

Comment:
@tomerd I have no idea, why `swiftformat` now removes the `internal` markers... Do we want to explicitly disable https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#redundantextensionacl?

Reply:
meh. dialing redundantExtensionACL sounds right to me

Reply:
https://github.com/swift-server/swift-aws-lambda-runtime/pull/197

Reply:
```suggestion
    internal struct RuntimeClient {
```